analyzer/src/funTest/assets/projects/synthetic/godep/glide/src/main.go

```diff
@@ -0,0 +1,17 @@
+package main
```

Comment:
Any particular reason why you created this file in a "src" sub-directory instead of the root like in the other added synthetic project?

Reply:
Yes, it's a requirement from Glide that there is a src folder with a go file in it.

Reply:
Really? Because the original `sprig` does not contain a `src` sub-directory...

Reply:
Weird, yesterday it failed with an error message until I moved the file to the src directory. Maybe because I had `GOPATH` set from another test before. Anyway, changed it now.